test-requirements.in

```diff
@@ -0,0 +1,12 @@
+flake8
+flake8-docstrings
+pytest
+pytest-cov
+pytz
+mock
+tox
```

Comment:
Could you please explain why so many dependencies were added here compared to the way it was before?

Reply:
I think I had problems building and running image and that's why all these(and others dependencies that you mentioned) were added.